server/app/Listeners/UserVerificationRequestNotification.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Listeners;
+
+use App\Models\User;
+use App\Models\Role;
+use App\Events\UserVerificationRequest;
+use App\Notification\Notifier;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use URL;
+
+
+class UserVerificationRequestNotification implements ShouldQueue
+{
+    /**
+     * Create the event listener.
+     *
+     * @return void
+     */
+    public function __construct(Notifier $notifier)
+    {
+        $this->notifier = $notifier;
+    }
+
+    /**
+     * Handle the event.
+     *
+     * @param  UserVerificationRequest $event
+     * @return void
+     */
+    public function handle(UserVerificationRequest $event)
+    {
+        $fname = $event->user->fname;
+        $lname = $event->user->lname;
+        $email = $event->user->email;
+        $url = URL::to('administrative-dashboard/verify-account');
+
+        $text = "$fname $lname ($email) is requesting account verification.";
```

Comment:
Instead of `$fname $lname` should we use `$user->display_name`? It currently just slaps the two together, but it could do something different in the future. This isn't necessarily for "display", but seems a little better than putting the two parts together by hand.